static/app/components/profiling/flamegraph/callTreeTable.tsx

```diff
@@ -0,0 +1,505 @@
+import {forwardRef, Fragment} from 'react';
```

Comment:
All of the code in this file is existing code that has moved and renamed as the old "FrameCallers" naming was confusing.